### ISSUE TITLE
Performance metrics for experiments

### ISSUE THREAD

**Issue by BrianWhiteaker** [BrianWhiteaker/HMC#5]

**Comment by BrianWhiteaker** [comment]
Max squared jump distance
Convergence time
Empirical- Bayesian Fraction of Missing Information (E-BMFI)
MCMC-Standard Error (MCMC-SE)  --> uses Effective Sample Size ESS

**Comment by BrianWhiteaker** [comment]
Split $\hat{R}$ is considered a strong diagnostic.

**Comment by BrianWhiteaker** [comment]
Comparing accept rate between random walk MC, MCMC, adaptive step HMC would be good for the paper.

Also steps to bypass a pinch in typical set.

**Comment by j2schmit** [comment]
Agreed, and I would include vanilla HMC in that mix. Also, it will be good to compare the # of gradient evaluations needed for a fixed number of samples between HMC and adaptive HMC. We'll probably also need to add NUTS into the comparison, or the simpler algorithm that Hoffman has more recently proposed, which I think uses more of the metric syou've outlined above.

**Comment by BrianWhiteaker** [comment]
https://ucsd.zoom.us/j/94922506338

**Comment by j2schmit** [comment]
I've attached the first paper (actually tied for first) that demonstrated
how to construct variable step size adaptive integrators. In particular
check out equations (11) and (12). These are formulas for the arc-length
parameterization as the monitor function.

**Comment by j2schmit** [comment]
Here's the paper (by Sanz-Serna) referenced in the Hairer paper that
proposes the arc-length parameterization

**Comment by j2schmit** [comment]
Ok, that last Sanz-Serna paper is a pain to read and not as directly
translatable to our use case. This HUang & Leimkuhler paper is much better
for what we need.

**Comment by BrianWhiteaker** [comment]
Sounds good
